guides/v2.1/ext-best-practices/tutorials/dynamic-row-system-config.md

```diff
@@ -0,0 +1,79 @@
+---
+group: extension-best-practices
+subgroup: 02_Extension-Coding
+title: Creating a dynamic row system config
+menu_title: Creating a dynamic row system config
+menu_order: 1010
+functional_areas:
+  - Standards
+---
+
+### Overview
+
+This tutorial gives instructions for adding a new dynamic rows system config in the {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento admin{% endglossarytooltip %}, by extending the [Magento/Config/Block/System/Config/Form/Field/FieldArray/AbstractFieldArray][0]{:target="_blank"} class. 
+
+## Step 1: Add your new system field 
+{:#step-1}
```

Comment:
Why are these here for the steps?

`{:#step-1}`, `{:#step-2}`?

Reply:
I was checking another example (in order to show them as steps in the right sidebar), but most probably they are old versioned, so removed them.
Thank you.